src/aviaryRules/aviaryNoClashingBoxColorProps/aviaryNoClashingBoxColorProps.js

```diff
@@ -0,0 +1,37 @@
+const meta = {
+  type: "problem",
+  docs: {
+    description: "Prevents isColor and accentColor from being passed to the same Box component",
+    category: "no-clashing-box-color-props",
+    recommended: false,
+  },
+  fixable: null,
+  schema: [],
+};
+
+const create = context => {
+  return {
+    JSXOpeningElement: node => {
+      if (node.name.name !== "Box") return;
+
+      const attributes = node.attributes;
+
+      const isColor = attributes.find(
+        attr => attr.type === "JSXAttribute" && attr.name.name === "isColor"
+      );
+      const accentColor = attributes.find(
+        attr => attr.type === "JSXAttribute" && attr.name.name === "accentColor"
+      );
```

Comment:
I just updated the code to prevent it from failing on spread operators

Note: It will still throw an error for conditional logic like this, but I'm thinking we keep it simple for now, and cater for this scenario if it becomes common

```
isColor={isTrue ? undefined : "system"}
accentColor={isTrue ? "forest" : undefined}
```

Reply:
I think that's fine tbh. The odds of that happening are probably pretty slim like you said. 
In any case, people can workaround that problem by disabling the rule or writing something like so:

```tsx
const props = {};

if (isTrue) {
  props[accentColor] = "forest"; 
} else {
  props[isColor] = "system";
}

<Box {...props}>
```

Not super pretty, probably better to use the disable comment but I like to actively discourage that habit.